src/fcc_bridge_node_mavsdk.cpp

```diff
@@ -0,0 +1,274 @@
+//
+// Created by Johan on 30.04.2024.
+//
+
+#include "fcc_bridge_node.hpp"
+
+// Libc header
+#include <cinttypes>
+#include <map>
+
+namespace fcc_bridge {
+namespace {
+// Path to uart device on Raspberry PI
+constexpr char const *UART_DEVICE_PATH = "/dev/serial0";
```

Comment:
This depends how we pass it through to the docker, correct? We should make sure to pass it through like this then as well.

Reply:
Absolutely. This is the path to default UART device on the PI. But for all intents and purposes this is only a placeholder.